articles/choosing-between-firebase-and-mongodb/index.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: choosing-between-firebase-and-mongo-db
+title: Choosing Between Firebase and MongoDB
+description: Respective history and pros and cons of using Firebase or MongoDB in a tech stack.
+date: 2020-11-09T00:00:00-07:00
+topics: [databases][mongodb][firebase]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/choosing-between-firebase-and-mongo-db/hero.jpg
+    alt: Relational Databases
+---
+
+### MongoDB vs Firebase
+Databases play an important role in determining the functionality and quality of an application or a website. For an app to be successful,
+the developer will have to consider the users of the application or the website.
+The quantity of data stored and retrieved from the application is one of the factors considered when choosing a database.
+Other factors considered when choosing the right database for an application or a website are:
+1. Latency.
+Latency refers to the response time of the database and to the end-to-end response time of the application. 
+2.structure of the data. Choosing the right database for an application or a website involves the developer understanding the right data structures for storage and retrieval of data. Various applications deal with data of various formats In this article, I am going to focus on Firebase and MongoDB while highlighting its differences.
+### History of MongoDB
+10gen company which is now MongoDB Inc developed MongoDB in the year 2007. The 10gen main language was server-side JavaScript.10gen company started to develop as a Platform as a service ( PAAS) product, but they experienced scalability issues with the existing relational database systems. They developed a document-oriented database system called MongoDB.
```

Comment:
This doesn't make sense. You are trying to say that 10gen was the original company that developed MongoDB in 2007 and changed their name.

"10gen company" is not English. => "The company 10gen"

Capitalize Platform as a Service
Spacing around (PAAS)

What were the existing relational database systems?

What does document-oriented mean?

Reply:
Why and how did they develop MongoDB? You can go into the origin story and cite your sources.